database/mysql/relresolver.go

```diff
@@ -0,0 +1,26 @@
+package mysql
+
+import (
+	"errors"
+
+	"github.com/spf13/viper"
+)
+
+// ConfigReader ...
+type ConfigReader struct {
+	v *viper.Viper
+}
+
+func (g *ConfigReader) readPrimaryRecord() (pRecordType string, err error) {
+	c := g.v.Sub("primary_record_type")
+	pRecordType = c.AllKeys()[0]
```

Comment:
Why make it a whole section, but then only read the first key?

Reply:
Ah, this probably needs some explanation in the code then :)

In the future, we want to be able to allow users to specify multiple record types and specify how many of each they want. Imagine that the configuration looks like this?
```
[record_types]
"customers" = 1000
"orders" = 5
```
I don't think we want to deal with that level of complexity yet. So for now, a user will only specify a primary_record_type,
```
[primary_record_type]
"customers" = 1000
```
The benefit of starting the config out as a section means in the future, we don't need to change the config format to accommodate the multiple-record-types change.